addons/sale_loyalty/models/sale_order.py

```diff
@@ -175,6 +190,20 @@ def action_open_reward_wizard(self):
             return True
         return self.env['ir.actions.actions']._for_xml_id('sale_loyalty.sale_loyalty_reward_wizard_action')
 
+    def action_view_gift_cards(self):
+        self.ensure_one()
+        return {
+            'name': "Gift Cards",
+            'type': 'ir.actions.act_window',
+            'view_mode': 'list,form',
+            'res_model': 'loyalty.card',
+            'domain': [
+                ('order_id', '=', self.id),
+                ('program_type', '=', 'gift_card'),
+            ],
```

Comment:
```suggestion
            'domain': [('order_id', '=', self.id), ('program_type', '=', 'gift_card')],
```
less then 100 in one line :thinking:

Reply:
I did it for better readability, hoping future additions would be easier. Nonetheless, I would change it. 😊